todo_app/templates/index.html

```diff
@@ -11,8 +11,21 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items  %}
+        <li> Id: {{item.id}} Title: {{ item.title}} Status: {{item.status}}</li>
+        <form  action="/deleteitem" method ='POST'>
+          <input type =text name= "id" value = "{{item.id}}"/>
+         
+          <input type= "submit" value = "Delete"/>
+        </form>
+        {% endfor%}
       </ul>
+      <form  action="/additem" method ='POST'>
+        <input type =text name= "title"/>
```

Comment:
There are other useful attributes you might want. E.g. this would mean the form cannot be submitted with a blank title

```suggestion
        <input type="text" name="title" required/>
```